Exercise_1.py

```diff
@@ -3,20 +3,43 @@
   
 # It returns location of x in given array arr  
 # if present, else returns -1 
-def binarySearch(arr, l, r, x): 
-  
+def binarySearch(arr, l, r, x):
```

Comment:
This is not correct binary search implementation. Please refer the pinned notes on the channel. You can also contact me, the TA or your professor.